src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/CoreNode.cs

```diff
@@ -212,6 +214,26 @@ private void StartBitcoinCoreRunner()
                 failureReason: $"Failed to invoke GetBlockHash on BitcoinCore instance after {duration}");
         }
 
+        private void StartStratisXRunner()
```

Comment:
interesting, these 2 methods `StartBitcoinCoreRunner` and `StartStratisXRunner` don't actually start the nodes, but rather wait for them to be up. the starting is done is `Start()`.
Can we merge them into 1 method (they're really the same) and call it somethingmore appropriate?
Then in `Start()` we'll have `if (this.runner is BitcoinCoreRunner || this.runner is StratisXRunner) WaitUntilRunning()` (please find a better name lol).